### PR TITLE
restricted product page added

### PR DESCRIPTION
**Pull request by saravanpa-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/7961

*Description of changes:*

added restricted access page

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/tests/unit/api/test_bulk_user.py [hunk 1]
```diff
@@ -20,7 +20,6 @@
 import uuid
 from random import randint
 
-import pytest
```
Comment (saravanpa-aot) [inline auth-api/tests/unit/api/test_bulk_user.py:23]: unused flake8 error

File: auth-api/tests/unit/api/test_org.py [hunk 2]
```diff
@@ -585,7 +585,6 @@ def test_update_org_duplicate_branch_name(client, jwt, session, keycloak_mock):
     assert rv.status_code == http_status.HTTP_409_CONFLICT
 
 
-
```
Comment (saravanpa-aot) [inline auth-api/tests/unit/api/test_org.py:588]: again flake8 error

**Review by karthik-aot — APPROVED** [review]

File: auth-web/src/views/auth/RestrictedProductView.vue [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  @Prop({ default: '' }) private orgId: string
+  @Prop({ default: 'Wills Registry application' }) private productName: string
+
+  private get productUrl ():string {
+    return `/account/${this.orgId}/settings/product-settings`
+  }
+}
+</script>
+
+<style lang="scss" scoped>
```
Comment (karthik-aot) [inline auth-web/src/views/auth/RestrictedProductView.vue:33]: we can remove this <style>

Comment (saravanpa-aot) [reply auth-web/src/views/auth/RestrictedProductView.vue]: done

File: auth-api/src/auth_api/schemas/user_settings.py [hunk 4]
```diff
@@ -24,7 +24,8 @@ class UserSettingsSchema(ma.ModelSchema):  # pylint: disable=too-many-ancestors,
     class Meta:  # pylint: disable=too-few-public-methods
         """Maps all of the User Settings fields to a default schema."""
 
-        fields = ('id', 'label', 'urlorigin', 'urlpath', 'type', 'account_type', 'account_status', 'product_settings')
+        fields = ('id', 'label', 'urlorigin', 'urlpath', 'type', 'account_type', 'account_status', 'product_settings',
```
Comment (sumesh-aot) [inline auth-api/src/auth_api/schemas/user_settings.py:27]: no need to add a new entry, instead pass the new url as product_settings

File: auth-web/src/routes/router.ts [hunk 5]
```diff
@@ -523,6 +524,13 @@ export function getRoutes (): RouteConfig[] {
       props: true,
       meta: { requiresAuth: false }
     },
+    {
+      path: '/account/:orgId/restricted-product/:productName?',
+      name: 'RestrictedProduct',
+      component: RestrictedProductView,
+      props: true,
+      meta: { requiresAuth: false }
```
Comment (sumesh-aot) [inline auth-web/src/routes/router.ts:532]: requiresAuth needs to be true

File: auth-web/src/views/auth/RestrictedProductView.vue [hunk 6]
```diff
@@ -0,0 +1,31 @@
+<template>
```
Comment (sumesh-aot) [inline auth-web/src/views/auth/RestrictedProductView.vue:1]: may be add a simple test class too ?

**Review by sumesh-aot — COMMENTED** [review]
please see comments

**Review by shabeeb-aot — APPROVED** [review]

File: auth-web/src/views/auth/RestrictedProductView.vue (hunk 6 above)
Comment (saravanpa-aot) [reply auth-web/src/views/auth/RestrictedProductView.vue:1]: done

File: auth-web/src/routes/router.ts (hunk 5 above)
Comment (saravanpa-aot) [reply auth-web/src/routes/router.ts:532]: yup..changed it

File: auth-api/src/auth_api/schemas/user_settings.py (hunk 4 above)
Comment (saravanpa-aot) [reply auth-api/src/auth_api/schemas/user_settings.py]: changed

**Review by sumesh-aot — APPROVED** [review]

**Pull request merged**